### ISSUE TITLE
Node-firebird don't give results if more than 3 fields in the query

### ISSUE THREAD

**Issue by wakeupwakeup** [hgourvest/node-firebird#216]
I'm testing "node-firebird" with firebird 2.5 and it runs if I query for 1, 2 or 3 fields, but If I ask for a fouth field it doesn't give any response. It is not a problem with any exact field because it querys well if I ask for it with less other fields. Depending on the table the number of fields I can query is different. Thanks

```
var Firebird = require('node-firebird');

var options = {};
 
options.host = '127.0.0.1';
options.port = 3050;
options.database = 'xxx.FDB';
options.user = 'SYSDBA';
options.password = 'masterkey';
options.lowercase_keys = false; // set to true to lowercase keys
options.role = null;            // default
options.pageSize = 4096;        // default when creating database


Firebird.attach(options, function(err, db) {

    if (err)
        throw err;


    db.query('SELECT fiel1, field2, field3_act FROM table', function(err, result) {
 
        console.log("result", result)

        db.detach();
    });

 
});
```

**Comment by mrotteveel** [comment]
This question was also crossposted to https://stackoverflow.com/questions/63032898/node-firebird-dont-give-results-if-more-than-3-fields-in-the-query

**Comment by mrotteveel** [comment]
This could be a duplicate of #209

**Comment by maratth** [comment]
You can give your table structure ?

**Comment by rafaelkerni** [comment]
I have a similar problem.
With an SQL that returns less than 30 rows the _query_ function enter to callback, more of this no.
I found a provisory solution using _.on('result'_


    fb.attach(config, (err, db) => {
        if (err)
            throw err;

        db.on('result', function(result) {
            console.log('result', result);
        });

        db.on('error', function(err) {
            console.log('err', err)
        });

        db.query(`select C.CODCLIENTE
                from Z_CLIENTES C
                where C.DATAINT >= '20.07.2020 00:00:00'` , (err, result) => {
            console.log('not entering here!')
        })

        db.detach();
    })

**Comment by mreis1** [comment]
@rafaelkerni your solution won't work if you have to many records to fetch since db.detach() will be called before you receive all the records on 'result' event.

**Comment by chandero** [comment]
I have a similar problem. I just change DEFAULT_FETCHSIZE from 200 to 10 and this work, but I do not know why this happend.
excuse my poor english.
